imports/plugins/core/core/server/methods/getOpaqueIdFromInternalId.js

```diff
@@ -1,3 +1,4 @@
+import { check } from "meteor/check";
```

Comment:
HI Erik, hat's this for?

Reply:
This is a meteor helper that checks types of input to make sure the data you receive matches what you are expecting. Kind of similar to a `PropType` check in React.

Meteor magically imports a lot of the core functionality, so that's why this was not imported before, it was just being done magically in the background. We are trying to get away from that and purposefully import everything needed, which is why this error was thrown.